scripts/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# build stage
+FROM archlinux/base AS builder
+
+# install build dependencies
+RUN pacman --noconfirm -Sy \
+    gcc \
+    git \
+    rust \
+    diffutils \
+    file \
+    awk \
+    pkg-config \
+    make
+
+WORKDIR /tmp/tox
+
+# download tox-node
+RUN git clone https://github.com/tox-rs/tox-node.git
```

Comment:
This is not quite right if we are going to build each PR. We have to build the current source which is local code in docket hub. So you should COPY . /tox-node and build it without cloning the repo two times

Reply:
I think, we might build on each merging PR into master.
docker hub automatically copy source from master, this line might be deleted.